packages/palette/src/elements/EntityHeader/EntityHeader.ios.tsx

```diff
@@ -39,14 +39,9 @@ export const EntityHeader: SFC<EntityHeaderProps> = ({
         </Flex>
       )}
       <Flex justifyContent="center" width={0} flexGrow={1}>
-        <Serif
-          ellipsizeMode="tail"
-          numberOfLines={1}
-          size="3t"
-          color="black100"
-        >
+        <Sans ellipsizeMode="tail" numberOfLines={1} size="3t" color="black100">
```

Comment:
Based on the [figma file](https://www.figma.com/file/8HxcgPD6duYjJOGO3R6r4t/Collection-Hubs?node-id=363%3A12621), I believe this should also be changed to be `3` instead of `3t`!

Other than that this looks 👍!

![image](https://user-images.githubusercontent.com/2081340/82908468-076f3380-9f36-11ea-926b-c04f1572d2a4.png)